examples/guidelines.md

```diff
@@ -0,0 +1,118 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+# Example Guidelines
+
+## An Example Is
+
+An example demonstrates running an application/framework/workload on
+Kubernetes in a meaningful way. It is educational and informative.
+
+Examples are not:
+
+* Full app deployments, ready to use, with no explanation. These
+  belong either
+  [here](https://github.com/kubernetes/deployment-manager) or in
+  something like [Helm](https://helm.sh/).
+* Simple toys to show how to use a Kubernetes feature. These belong in
+  the [user guide](../docs/user-guide/).
+* Demos that follow a script to show a Kubernetes feature in
+  action. Example: killing a node to demonstrate controller
+  self-healing.
+* A tutorial which guides the user through multiple progressively more
+  complex deployments to arrive at the final solution. An example
+  should just demonstrate how to setup the correct deployment
+
+## An Example Includes
+
+### Up front
+
+* Has a "this is what you'll learn" section.
+* Has a Table of Contents.
+* Has a section that brings up the app in the fewest number of
+  commands (TL;DR / quickstart), without cloning the repo (kubectl
+  apply -f http://...).
+* Points to documentation of prerequisites.
+  * [Create a cluster](../docs/getting-started-guides) (e.g., single-node docker).
+  * Setup kubectl.
```

Comment:
Wherever https://github.com/kubernetes/kubernetes/blob/master/docs/user-guide/prereqs.md moves to on the new website.